en/02_Developer_Guides/00_Model/08_SQL_Select.md

```diff
@@ -54,15 +54,17 @@ various assumptions the ORM and code based on it have:
 We'll explain some ways to use the low-level APIs with the full power of SQL,
 but still maintain a connection to the ORM where possible.
 
-[warning]
-Please read our [security topic](/developer_guides/security) to find out
-how to properly prepare user input and variables for use in queries
-[/warning]
+> [!WARNING]
+> Please read our [security topic](/developer_guides/security) to find out
+> how to properly prepare user input and variables for use in queries
 
 ## Usage
 
 ### Getting table names
 
+> [!WARNING]
+> Because of the way the ORM interacts with class inheritance, some models will spread their data across multiple tables. See [Joining tables for a DataObject inheritance chain](#joins-for-inheritance) below for information about how to handle that scenario.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.